example/grafana/grafana.ini

```diff
@@ -0,0 +1,5 @@
+instance_name = "grafana"
+
+[security]
+admin_user = admin
```

Comment:
I think it's better to set these as environment variables in the docker-compose rather than a file to avoid making an example of insecure values

https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables
`GC_SECURITY_ADMIN_USER` / `_PASSWORD`

Reply:
I would not overcomplicate it as this is just for example purposes and inside the scope of docker compose.

Reply:
Especially if moving the files it's better - though I don't know if it's particularly complicated using variables either :P